modules/apps/style-book/style-book-api/src/main/java/com/liferay/style/book/exception/StyleBookEntryThemeIdException.java

```diff
@@ -0,0 +1,27 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.style.book.exception;
+
+import com.liferay.portal.kernel.exception.PortalException;
+
+/**
+ * @author Anderson Luiz
+ */
+public class StyleBookEntryThemeIdException extends PortalException {
+
+	public static class MustNotBeNull extends StyleBookEntryThemeIdException {
+
+		public MustNotBeNull() {
+			super("Value must not be null");
```

Comment:
When defining the exception message, you should use the property name/column name. Then, you should replace the word `Value` for `Theme ID`.
Can you rewrite it please?

`super("Theme ID must not be null");`